hyper/http11/connection.py

```diff
@@ -137,6 +144,33 @@ def connect(self):
 
         return
 
+    @classmethod
+    def _create_tunnel(cls, proxy_host, proxy_port, target_host, target_port):
+        """
+        Sends CONNECT method to a proxy and returns a socket with established
+        connection to the target.
+
+        :returns: socket
+        """
+        conn = cls(proxy_host, proxy_port)
+        conn.connect()
+        conn._send_headers(to_bytestring('CONNECT'),
```

Comment:
As this is a constant, this can just be `b'CONNECT'`. We only need `to_bytestring` to allow string formatting operations.